db/migrate/20231118153202_create_likes.rb

```diff
@@ -0,0 +1,8 @@
+class CreateLikes < ActiveRecord::Migration[7.1]
+  def change
+    create_table :likes do |t|
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] Kindly add reference to the post and user table in the like table to reference the likes relationship with the users and posts in the ERD diagram. Example snippet:

```ruby
     create_table :likes do |t|
    t.timestamps # this will crate uopdated_at and created_at columns
    t.references :user, foreign_key: true # this will create user_id and a corresponding index  (but students can also add column and index in separate commands)
    t.references :post, foreign_key: true # this will create post_id and a corresponding index (but students can also add column and index in separate commands)
  end

```

Reply:
well resolved! ✔️